modules/sso/jetpack-sso-login.js

```diff
@@ -0,0 +1,25 @@
+jQuery( document ).ready( function( $ ) {
+	var ssoWrap = $( '.jetpack-sso-wrap' ),
```

Comment:
I would rather not address changing the SSO button class for now, as that [would likely require string changes](https://github.com/Automattic/jetpack/blob/master/modules/sso.php#L842). It's something I will keep in mind as I continue to refactor SSO.